decks/emoji.mdx

```diff
@@ -0,0 +1,98 @@
+import { CodeSurfer as Surfer } from "code-surfer";
+import { CodeSurferColumns, Step } from "code-surfer";
+import { Appear, Background } from "gatsby-theme-mdx-deck";
+import * as L from "../src/layout";
+import customTheme from "../src/theme";
+import GreetingLoader from "./src/greeting-loader";
+
+export const theme = customTheme;
+
+# Workshop Proposal: Emoji Sentiment Data
+ 
+---
+ 
+import { CodeSurfer as Surfer } from "code-surfer";
+import { CodeSurferColumns, Step } from "code-surfer";
+import { Appear, Background } from "gatsby-theme-mdx-deck";
+import * as L from "../src/layout";
+import customTheme from "../src/theme";
+import GreetingLoader from "./src/greeting-loader";
+ 
+export const theme = customTheme;
+ 
+# Workshop Proposal: Emoji Sentiment Data
+
+ 
+---
+
+# Specific Idea
+Ideas:
+1. predicting a tweet's emotion based on emojis used
+- predicting a tweet’s emotion -> generalize to predict general attitude from multiple tweets on a specific event? (tracking hashtags) / predict someone’s general feeling (in a week/month/year) by analyzing emojis in a person’s tweets.
+- what tweets are we analyzing?
+2. words to emojis: emotion translator (wrote about this below!)
+ 
+---

```

Comment:
What are emoji's? What is twitter? Including images might help the reader understand the context better.